docs/language/src/builtins/widgets.md

```diff
@@ -232,6 +232,32 @@ export component Example inherits Window {
 }
 ```
 
+## `ProgressIndicator`
+
+Indicates a progress of a process or an operation.
+
+### Properties
+
+-   **`indeterminate`**: (_in_ _bool_): Defaults to false. Displays an animation instead of the value if set to true.
+-   **`value`** (_in-out_ _float_): The value.
```

Comment:
```suggestion
-   **`value`** (_in-out_ _float_): The progress value of the operation, between `minimum` and `maximum`.
```

I'm curious, btw, why is it `in-out` instead of just `in`?